test/rest.test.js

```diff
@@ -1184,6 +1184,33 @@ describe('strong-remoting-rest', function() {
           .expect(500)
           .end(expectErrorResponseContaining({message: 'an error'}, done));
       });
+      
+      it('should return 500 for unhandled errors', function(done) {
+        var method = givenSharedStaticMethod(
+          function greet(name, cb) {
+            cb(null, name );
+          },
+          {
+            accepts: { arg: 'name', type: 'string', http: {source: 'body'} },
+            returns: { arg: 'name', type: 'string'}
+          }
+        );
+      
+        objects.before(method.name, function(ctx, next) {
+          process.nextTick(next);
+        });
+       
+        objects.before(method.name, function(ctx, next) {
+          throw new Error('test error');
+        });       
+       
+        request(app).get(method.url)
+          .set('Accept', 'application/json')
+          .set('Content-Type', 'application/json')
+          .send({name: 'abc'})
```

Comment:
In the same spirit as above, remove `Content-Type` and request body (`.send()`) as they are not necessary.